go-controller/pkg/node/node.go

```diff
@@ -256,6 +261,9 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	// start health check to ensure there are no stale OVS internal ports
 	go checkForStaleOVSInterfaces(n.stopChan)
 
+	// start health check to ensure there are no stale OVS internal ports
```

Comment:
comment looks wrong here. should be "// start the management port health check"

Reply:
fixed